src/test/java/uk/gov/pay/ledger/queue/sqs/EventQueueIntegrationTest.java

```diff
@@ -0,0 +1,70 @@
+package uk.gov.pay.ledger.queue.sqs;
+
+import com.fasterxml.jackson.core.JsonProcessingException;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import org.junit.Ignore;
+import org.junit.Test;
+import uk.gov.pay.ledger.app.LedgerConfig;
+import uk.gov.pay.ledger.app.config.QueueMessageReceiverConfig;
+import uk.gov.pay.ledger.app.config.SqsConfig;
+import uk.gov.pay.ledger.event.model.Event;
+import uk.gov.pay.ledger.queue.EventQueue;
+import uk.gov.pay.ledger.queue.QueueException;
+import uk.gov.pay.ledger.rules.SqsTestDocker;
+
+import static org.hamcrest.core.Is.is;
+import static org.junit.Assert.assertFalse;
+import static org.junit.Assert.assertThat;
+import static org.mockito.Mockito.mock;
+import static org.mockito.Mockito.when;
+import static uk.gov.pay.ledger.utils.fixtures.EventFixture.anEventFixture;
+
+@Ignore
+public class EventQueueIntegrationTest {
```

Comment:
I think we have got different naming conventional for integration tests. Lets us move to recent naming convention introduced recently for java apps and to be `EventQueueIT`